etl/steps/data/meadow/un/2024-10-21/census_dates.py

```diff
@@ -0,0 +1,33 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap = paths.load_snapshot("census_dates.csv")
+
+    # Load data from snapshot.
+    tb = snap.read()
+
+    #
+    # Process data.
+    tb = tb.drop_duplicates()
+    tb = tb[tb["Date"].apply(len) > 3]
```

Comment:
could be nice to explain why you are doing this filtering here. I am sure there is a good reason so maybe add a comment so it's easier to remember why you've done this next time you update the data?